server/vertx-server/src/main/scala/sttp/tapir/server/vertx/streams/ReadStreamCompatible.scala

```diff
@@ -20,3 +21,10 @@ object ReadStreamCompatible {
     override def fromReadStream(s: ReadStream[Buffer]): Nothing = ???
   }
 }
+
+trait VertxStreams extends Streams[VertxStreams] {
+  override type BinaryStream = ReadStream[Buffer]
+  override type Pipe[A, B] = Processor[A, B]
```

Comment:
I would go with ReadStream[A] => ReadStream[B] to not mix reactive streams and vertx streams

Reply:
done